docs/_build/.gitignore

```diff
@@ -1 +0,0 @@
-*
```

Comment:
Did you mean to delete this? (I noticed the files we're left un-tracked after running the tests)